### PR TITLE
Semaphore subgraph in main repository

### LINKED ISSUE #541: Move Semaphore subgraph to the main repository

### PR DESCRIPTION
**Pull request by cedoor**
## Description

The Semaphore subgraph has been moved to the main monorepo in the `apps` folder. 

## Related Issue

Closes #541 

## Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: .github/workflows/production.yml [hunk 1]
```diff
@@ -43,12 +43,48 @@ jobs:
...
+              id: yarn-cache
+              with:
+                  path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+                  key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-yarn-
+
+            - name: Install dependencies
+              run: yarn
+
```
Comment (0xjei) [inline .github/workflows/production.yml:81]: Don't we need to run the `yarn codegen` before the `yarn build` command?

File: apps/subgraph/tests/.bin/semaphore.wasm
Comment (0xjei) [inline apps/subgraph/tests/.bin/semaphore.wasm:1]: Why do we need the `wasm` here?

**Review by 0xjei — COMMENTED** [review]
Looks good, just wondering about two potential minors. Looking forward for your feedback @cedoor

File: .github/workflows/production.yml (hunk 1 above)
Comment (cedoor) [reply .github/workflows/production.yml:81]: We do, `yarn codegen` is run in the `build:subgraph` script of the root `package.json`.

Comment (0xjei) [reply .github/workflows/production.yml:81]: sweet, thank you!

File: apps/subgraph/tests/.bin/semaphore.wasm
Comment (cedoor) [reply apps/subgraph/tests/.bin/semaphore.wasm:1]: We can actually add that file to `.gitignore`. Let me update it.

**Review by 0xjei — APPROVED** [review]

**Review by vplasencia — APPROVED** [review]

**Pull request merged**